src/plugins/discover/public/application/components/sidebar/discover_field_bucket.tsx

```diff
@@ -70,14 +70,14 @@ export function DiscoverFieldBucket({ field, bucket, onAddFilter }: Props) {
                     ? emptyTxt
                     : `${bucket.display}: ${bucket.count} (${bucket.percent.toFixed(1)}%)`
                 }
-                size="xs"
+                size="s"
                 className="eui-textTruncate"
               >
                 {bucket.display === '' ? emptyTxt : bucket.display}
               </EuiText>
             </EuiFlexItem>
             <EuiFlexItem grow={false} className="eui-textTruncate">
-              <EuiText color="secondary" size="xs" className="eui-textTruncate">
+              <EuiText color="secondary" size="s" className="eui-textTruncate">
                 {bucket.percent.toFixed(1)}%
```

Comment:
Similarly, if this is in the data popover for the left pane, let's leave this small.

Reply:
will do

Reply:
Just to be clear, I mean xs here